perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/HierarchicalHashedDataGenerator.java

```diff
@@ -94,18 +99,19 @@ private GenDataNode getDataForResource(Resource resource) {
 		} else {
 			members = sess.getPerunBl().getResourcesManagerBl().getAllowedMembers(sess, resource);
 		}
+		allMembers.addAll(members);
 
 		dataProvider.loadResourceAttributes(resource, members, false);
 
 		List<String> resourceAttrHashes = dataProvider.getResourceAttributesHashes(resource, false);
```

Comment:
Variable ``resourceAttrHashes`` is not used.

Reply:
Variable removed.